roles/container-engine/cri-o/tasks/main.yaml

```diff
@@ -86,7 +86,7 @@
     - "fedora-modular"
 
 - name: Install cri-o packages with osttree
-  command: "rpm-ostree install {{ crio_packages|join(' ') }}"
+  command: "rpm-ostree install --idempotent {{ crio_packages|join(' ') }}"
```

Comment:
Without the `--idempotent` I encountered problems if the package has already be installed from a proceeding (unsuccessful) ansible run.

Reply:
There is a check
```
- name: Check if already installed
  stat:
    path: "/bin/crio"
  register: need_bootstrap_crio
  when: is_ostree
```
to avoid install and reboot. Can you elaborate what encountered problems are?

Reply:
Well, I think it has stroked me because of my working flow: I tried to get kube running, stepped upon a problem, tried to fix it _and then rerun the ansible scripts on the *old* VMs_.

There are several packages involved in installing crio and if one of the is already installed (for whatever reason) `rpm-ostree` will indicate an error without the `--idempotent`. I think `ansible` scripts should be as idempotent as possible.

However, at the end I restraint from the crio/podman variant (I have not find a way to get it running). Hence it is a problem, but wrong in this PR?!?

Reply:
no, just wondering why i never struggled with and i just want understand why it was not idempotent before. if need_boostrap_crio  == /bin/crio file exits, it will install and reboot, after rerun ansible script it should not try to reinstall it..